website/src/pages/use-cases/code-health.tsx

```diff
@@ -0,0 +1,359 @@
+                        </p>
+                    </div>
+                    <div className="text-center mb-4">
+                        <ClipBoardPulseOutlineIcon className="mb-4 text-blurple" size={40} />
+                        <h4 className="font-weight-bold">Monitor code health initiatives</h4>
+                        <p>
+                            Stay on top of changes to your code health. Monitor and measure code health initiatives and
+                            get actionable insights into the impact of large-scale changes.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-gradient-venus-radial">
+            <ContentSection className="my-7">
+                <div className="row flex-column flex-lg-row justify-content-between">
+                    <div className="p-lg-0 col-lg-6 px-4">
+                        <h1 className="mb-md-4 font-weight-bold max-w-500">Improving code health can be daunting</h1>
+                        <p className="max-w-500">
+                            Engineering teams need to track and measure code quality consistently to monitor code health
+                            across their entire codebase, but current tools don't make this easy.{' '}
+                            <strong>What does that mean for you?</strong>
+                        </p>
+                        <ul>
+                            <li className="mt-1">
+                                Old versions, libraries, or functions are littered throughout your code, resulting in
+                                incidents and backward compatibility issues.
+                            </li>
+                            <li className="mt-1">
+                                Unclear code ownership leads to unclear responsibility, making it hard for developers to
+                                find the right domain expert when they need help.
+                            </li>
+                            <li className="mt-1">
+                                Engineering managers struggle to justify prioritizing and addressing tech debt because
+                                success is difficult to track and measure.
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="col-lg-5 mt-lg-6 mt-5">
+                        <Blockquote
+                            quote={`With the help of Sourcegraph, we were able to quickly look at all clients of an API and remove unused attributes that lived in different repositories, ultimately simplifying our APIs and speeding up developer iteration time.`}
+                            by="Justin Phillips, Software Engineer at Lyft"
+                            logoImage="/external-logos/lyft-logo.svg"
+                            logoAlt="Lyft"
+                            border={true}
```

Comment:
This may be a good opportunity to refactor `<Blockquote>`. To match the spec better and follow our best practices, not passing style props and styling in line when possible. The spec has the side border go down past the author line. Using Bootstrap utility classes we can put in `border-left border-4 border-$theme-color`, style in-line and get rid of `case-studies__quote--in-content--section` as well.

Reply:
Agreed! I was able to take out most of `case-studies__quote--in-content--section`. It looks like we do not have `border-$width` because we are using Bootstrap 4.6 in `about`, so I left the 2px width in the class and renamed it `--border` for now.

Reply:
Right on! Looks awesome!! 🙌